test/functional/test_framework/messages.py

```diff
@@ -195,10 +195,6 @@ def FromHex(obj, hex_string):
     obj.deserialize(BytesIO(hex_str_to_bytes(hex_string)))
     return obj
 
-# Convert a binary-serializable object to hex (eg for submission via RPC)
```

Comment:
Non-blocking suggestion: Could add a comment to `FromHex` describing that the recommended inverse is `obj.serialize().hex()` (instead of a hypothetical `ToHex`) :)

Reply:
Good idea, I added a commit with your suggestion!